_test/private_assets_copy_team_images_test.rb

```diff
@@ -0,0 +1,72 @@
+# 18F Hub - Docs & connections between team members, projects, and skill sets
+#
+# Written in 2015 by Mike Bland (michael.bland@gsa.gov)
+# on behalf of the 18F team, part of the US General Services Administration:
+# https://18f.gsa.gov/
+#
+# To the extent possible under law, the author(s) have dedicated all copyright
+# and related and neighboring rights to this software to the public domain
+# worldwide. This software is distributed without any warranty.
+#
+# You should have received a copy of the CC0 Public Domain Dedication along
+# with this software. If not, see
+# <https://creativecommons.org/publicdomain/zero/1.0/>.
+#
+# @author Mike Bland (michael.bland@gsa.gov)
+
+require_relative "test_helper"
+require_relative "../_plugins/private_assets"
+require_relative "site"
+require_relative "temp_file_helper"
+
+require "minitest/autorun"
```

Comment:
I'm big on explicit, narrowly-scoped `require` statements in each file, to reduce dependencies and the chance that removing a dependency in one file will break another. Maybe that's paranoid on my part, but it helps me sleep at night! :-)